app/views/users.html

```diff
@@ -65,6 +66,7 @@ <h2><i class="glyphicon glyphicon-user"></i>&nbsp;&nbsp;Users</h2>
                       <span ng-if="user.user.locked" style="color: #d9534f" tooltip-html-unsafe="This user is currently locked<br />They will need to set their password or admin needs to set it for them"><i class="glyphicon glyphicon-lock"></i></span>
                     </td>
                     <td data-title="Name">{{ user.user.firstname }} {{ user.user.surname }}</td>
+                    <td data-title="Name">{{ user.user.provider }}</td>
```

Comment:
`data-title` should be different. "Name" is already taken

Reply:
nice done! thanks for the review